tools/devtool

```diff
@@ -441,6 +563,8 @@ cmd_help() {
     echo "        Performs prerequisites checks needed to execute firecracker."
     echo "    fix_perms"
     echo "        Fixes permissions when devtool dies in the middle of a privileged session."
+    echo "    build_devctr"
+    echo "        Builds a development docker container."
```

Comment:
nit: the description is a bit misleading, as this containers role is to build and test Firecracker - see the comments at the top of the file.

Reply:
Updated, let me know if the new one looks ok.